Source/Renderer/VertexArray.js

```diff
@@ -636,9 +673,28 @@ define([
         return this._attributes[index];
     };
 
+    // Work around for ANGLE, where the attribute divisor seems to be part of the global state instead
+    // of the VAO state. This function is called when the vao is bound/unbound, and should be removed
+    // once the ANGLE issue is resolved.
+    function setVertexAttribDivisor(vertexArray, enabled) {
```

Comment:
This could be more efficient, i.e., use fewer WebGL calls.  This could ping ping the divisor, e.g.,

bind: set divisor to 1
unbind: set to 0
bind: set to 1 // redundant

It should be straightforward to track the divisor per index, right?  We could also have a separate count of the number of instanced attributes so if it is zero for the previous draw call and the instanceCount for the current draw call is zero, we would skip the loop.